Packs/QuestKace/Integrations/QuestKace/QuestKace.py

```diff
@@ -0,0 +1,785 @@
+                                                                                                        'RamTotal',
+                                                                                                        'RamUsed'])
+    context = {
+        'QuestKace.Machine(val.ID === obj.ID)': context
+    }
+    return human_readable_markdown, context, raw_response
+
+
+def get_assets_list_command(client, args) -> Tuple[str, dict, dict]:
+    """Function which returns all assets in the system.
+        Args:
+            client : Integretion client which communicates with the api.
+            args: Users arguments of the command.
+       Returns:
+           human readable, context, raw response of this command.
+    """
+    limit = int(args.get('limit', 50))
+    filter_fields = args.get('custom_filter')
+    response = client.assets_list_request(filter_fields)
+    raw_response = response.get('Assets')[:limit]
+    context = parse_response(raw_response)
+    human_readable_markdown = tableToMarkdown('Quest Kace Assets', context, removeNull=True,
+                                              headers=['ID', 'Name', 'Created', 'Modified', 'OwnerID', 'MappedID',
+                                                       'AssetClassID', 'AssetDataID', 'AssetStatusID', 'AssetTypeID',
+                                                       'AssetTypeName'])
+    context = {
+        'QuestKace.Asset(val.ID === obj.ID)': context
+    }
+    return human_readable_markdown, context, raw_response
+
+
+def get_queues_list_command(client, args) -> Tuple[str, dict, dict]:
+    """Function which returns all queues in the system.
+        Args:
+            client : Integretion client which communicates with the api.
+            args: Users arguments of the command.
+       Returns:
+           human readable, context, raw response of this command.
+    """
+    filter_fields = args.get('custom_filter')
+    limit = int(args.get('limit', 50))
+    response = client.queues_list_request(filter_fields)
+    raw_response = response.get('Queues')[:limit]
+    context = parse_response(raw_response)
+    human_readable_markdown = tableToMarkdown('Quest Kace Queues', context, removeNull=True,
+                                              headers=['ID', 'Name', 'Fields'])
+    context = {
+        'QuestKace.Queue(val.ID === obj.ID)': context
+    }
+    return human_readable_markdown, context, raw_response
+
+
+def get_tickets_list_command(client, args) -> Tuple[str, dict, dict]:
+    """Function which returns all tickets in the system.
+        Args:
+            client : Integretion client which communicates with the api.
+            args: Users arguments of the command.
+       Returns:
+           human readable, context, raw response of this command.
+    """
+    limit = int(args.get('limit', 50))
+    custom_shaping = args.get("custom_shaping")
```

Comment:
is it listing by defaults from all queues? or just from one?
how will the shaping will behave in this case?(if there is no custom_shaping)

Reply:
By default the listing is from all queues.In the request it will check if shaping was given and if not it will use the shaping that was created by all queues.